packages/react-components/react-tooltip/src/components/Tooltip/renderTooltip.tsx

```diff
@@ -1,13 +1,20 @@
+/** @jsxRuntime classic */
+/** @jsxFrag React.Fragment */
```

Comment:
I noticed this `@jsxFrag` pragma is only in files that (currently) use fragments. Out of curiosity, what is the developer experience if you add a Fragment to a render___.tsx file that doesn't currently have a `@jsxFrag` pragma? Is there a build error? And/or is it clear what the fix is?

Reply:
Yes @behowell, there is a build error that is very explicit about the necessity of adding the `@jsxFrag` tag

<img width="778" alt="image" src="https://user-images.githubusercontent.com/5483269/232888403-fd999f34-a379-4b62-b296-efd267eaa1ab.png">

Let's keep it in mind that the whole jsx comment tags is a temporary solution in the meantime we're investigating how to properly implement those changes by our build configurations! https://github.com/microsoft/fluentui/pull/27475#discussion_r1165877963